lib/es-models/bib.js

```diff
@@ -101,32 +106,39 @@ class EsBib extends EsBase {
     let contributorLiterals = this._valueToIndexFromBasicMapping('contributorLiteral')
     if (!contributorLiterals) return null
 
-    // Filter out any contributors that also appear as creator:
-    const creators = this.creatorLiteral()
-    if (!creators) return contributorLiterals
-    contributorLiterals = contributorLiterals.filter((contrib) => {
+    contributorLiterals = contributorLiterals
+      .map(trimTrailingPeriod)
       // Contributor is redundant if there exists a creator of which the
```

Comment:
I think this comment can be removed

Reply:
I wanted some explanation about what is happening in that filter, so I simplified the comment.